cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/adls/AdlsStorageClient.java

```diff
@@ -64,4 +64,10 @@ public DataLakeFileClient getNativeClient() {
   public StorageType.Type getStorageType() {
     return ADLS_TYPE;
   }
+
+  @Override
+  public boolean pathExists(String path) {
+    // TODO: Support pathExists on ADLS
```

Comment:
Use exists()?
https://learn.microsoft.com/en-us/java/api/com.azure.storage.file.datalake.datalakepathclient?view=azure-java-stable#com-azure-storage-file-datalake-datalakepathclient-exists()

Reply:
Can we do this as part of separate PR? DLFC is not initialized: https://github.com/linkedin/openhouse/issues/148

Reply:
Sure

Reply:
nit: should we then throw not implement exception instead?